backend/app/services/chit_service.py

```
@@ -157,24 +158,80 @@ def _is_docked_mode(self) -> bool:
+            else:
+                # For development, allow unverified connections with warning
+                logger.warning(f"NATS CA certificate not found at {ca_file}, using default verification")
+
+            # Load client certificate if mutual TLS is configured
+            client_cert = os.getenv("NATS_TLS_CERT", "/app/nats-certs/client.crt")
+            client_key = os.getenv("NATS_TLS_KEY", "/app/nats-certs/client.key")
+            if os.path.exists(client_cert) and os.path.exists(client_key):
+                ctx.load_cert_chain(client_cert, client_key)
+                logger.info("Loaded NATS client certificate for mutual TLS")
+
+            return ctx
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)